### DISCUSSION TITLE
Simple turn based combat

### DISCUSSION THREAD

**Discussion by ryanmunger** [replit/kaboom#160] [Q&A]
I am very excited about this library. What a clean API!

I noticed the click action only passes the vector point. Is it possible to click on another character or player and have it locate or return the clicked on object?

This is to select a target in a very simple turn based combat system like final fantasy 1.

Thank for your help and I would love to share what I come up with!

- Ryan

**Comment by slmjkdbtl** [comment]
I'd give each clickable object with a tag like "character" and use [`clicks`](https://github.com/replit/kaboom/blob/e18f9d5292bd85c921cfb2d925a741d6112a9db8/src/types.ts#L164) (i just realized it's not in the docs, sorry)

```js
add([
    // ... normal comps
    "character",
]);

clicks("character", (obj: GameObj) => {
    // do stuff with obj
});
```

**Reply by ryanmunger** [reply]
@slmjkdbtl Awesome! Thank you for the quick response.
